src/app/(checkin)/checkin/scan/page.client.tsx

```
@@ -9,16 +9,19 @@ import React, {
   useImperativeHandle,
 } from 'react'
 import { QRScanner } from '@/components/QRScanner'
-import { MapPin, History, ChevronDown, Lightbulb, LightbulbOff, Upload } from 'lucide-react'
+import { History, ChevronDown, Upload } from 'lucide-react'
+import { usePathname } from 'next/navigation'
 import Link from 'next/link'
 import type { CheckinRecord, User } from '@/payload-types'
 import jsQR from 'jsqr'
+import { useTranslate } from '@/providers/I18n/client'
 
 const ScanHistory = forwardRef((props: {}, ref) => {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)